htdocs/luci-static/resources/view/homeproxy/client.js

```diff
@@ -250,6 +250,16 @@ return view.extend({
 		o.default = 'redirect_tproxy';
 		o.rmempty = false;
 
+		o = s.taboption('routing', form.Value, 'clashapi_port', _('Enable clash api'));
+		o.value('', _('Disable'));
+		o.value('9090', _('9090'));
+		o.validate = function(section_id, value) {
+			if (section_id && value && !stubValidator.apply('port', value)) {
+				return _('Expecting: %s').format(_('valid port value'));
+			}
+			return true;
+		}
```

Comment:
```suggestion
		o.datatype = 'port';
```

顺便，放 ipv6_support 下面吧